includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1963,10 +1963,17 @@ private function prepare_payment_information_from_request( WC_Order $order ) {
 			'selected_payment_type'         => $selected_payment_type,
 			'payment_method_types'          => $payment_method_types,
 			'shipping'                      => $shipping_details,
-			'statement_descriptor'          => $this->get_statement_descriptor( $order, $selected_payment_type ),
 			'token'                         => $token,
 			'return_url'                    => $this->get_return_url_for_redirect( $order, $save_payment_method_to_store ),
 		];
+
+		// Use the dynamic + short statement descriptor if enabled and it's a card payment.
+		$is_short_statement_descriptor_enabled = ! empty( $this->get_option( 'is_short_statement_descriptor_enabled' ) ) && 'yes' === $this->get_option( 'is_short_statement_descriptor_enabled' );
```

Comment:
Nit: I notice that this is a copy of the code below but checking if the value is empty and then if it's equal to `'yes'` seems redundant.

Reply:
Given this is a copy and paste, feel free to leave it if you want.